OpenShift/03_create_cluster.sh

```diff
@@ -72,7 +72,7 @@ function rhcos_image_url() {
 function get_provision_if() {
   #Dont do anything if there is a value already set
   if [[ -z "${INTERNAL_NIC}" ]]; then
-    lshw -quiet -class network | grep -A 1 "bus info" | grep name | awk -F': ' '{print $2}'|grep e | while read interface; do
+    for interface in $( sudo lshw -quiet -class network | grep -A 1 "bus info" | grep name | awk -F': ' '{print $2}'|grep e | tr '\n' ' '); do
```

Comment:
So I've run into this now,   but am confused as why you put said you needed the tr?   Also if you are going to want to remove any , wouldnt' you just to a tr -d '\n'?

Reply:
OK, seems it was specific to my env, removed _tr_ part